src/rest/encounterRest.js

```diff
@@ -44,6 +44,16 @@ const api = {
       }
     }),
 
+  // used in lab-workflow
+  getPatientEncounters: ({ patientUUID, encounterTypeUUID }) => axiosInstance.get(`encounter?s=default&patient=${patientUUID}&encounterType=${encounterTypeUUID}&v=full`)
```

Comment:
@rhenshaw56 I'm not sure exactly how this differs from the fetchEncountersByPatient besides the "s=default" and the representation type (the v parameter). (I'm not sure if s=default actually does anything?)  What were the problems you were encountering when you tried using the fetchEncountersByPatient method?  

If you need  a different REST representation than what is provided by DEFAULT_ENCOUNTER_REP I think you can either add additional parameters to DEFAULT_ENCOUNTER_REP or have fetchPatientByEncounters accept a third (optional) parameter called representation that allows one you override the DEFAULT_ENCOUNTER_REP value.

Reply:
@mogoodrich when I tried using the `fetchEncountersByPatient` rest call, I was getting this on postman.
<img width="1080" alt="screen shot 2018-10-24 at 16 49 54" src="https://user-images.githubusercontent.com/25608386/47443794-05b71a00-d7ad-11e8-9823-2c8ab2d7fc63.png">

Reply:
thanks @rhenshaw56 , can you include the rest of the error?

Reply:
It's possible I broke something changing the DEFAULT_REST_REP...